docs/business.yaml

```diff
@@ -4200,18 +4200,18 @@ components:
             - XP
             - XS
           example: 'GP: General Partnership'
-          description: 'Type of business eg: individual(SP), partnership(GP).'
+          description: 'Type of business eg: BC corporation (B), Continued in corporation (C), Sole proprietor(SP), partnership(GP).'
```

Comment:
Need to capitalize `P` of `partnership`?

Reply:
it's not capitalized in Dylan's doc, not sure if it's intended
![image](https://github.com/user-attachments/assets/98da77a5-5215-4a0b-a297-fcc02ab7113a)

Reply:
Good catch.

Will check with Dylan on both.

Reply:
Checked with Dylan and updated to General Partnership(GP)